Radio test project (#280)
DotBots/DotBot-firmware: radio_test/README.md

> # GOAL: Check the radio communication between two nRF52840DK

Reviewer: The radio test can work on any nrf52 and nrf5340. It's not limited to nrf52840dk

```suggestion
# GOAL: Check the radio communication between two nRF boards
```

## Segger:
  * **UnZIP** then open `DotBot-firmware` file and open project called `nRF52840dk` with Segger
  * Tx: **build** then **load** `projects_radio_test -> project Tx` on the first nRF
  * Rx: **build** then **load** `projects_radio_test -> project Rx` on the second nRF

>   * Tx_Idle: **build** then **load** `projects_radio_test -> project TxIdle` on the third nRF (you can change the power for your test)

Reviewer: I don't think the projects are still called this way

  ![Démo](../doc/sphinx/_static/images/radio_test_setup.jpg)
    
 ## Python /Conda:
  * **download** and **install** Anaconda 
  * **download** in the python file in **dist/scripts/radio_test**

>   * With Conda: `conda install anaconda::pyserial` (go next if already done)

Reviewer: conda is not a standard Python setup. People can use it of course.
It's not meaningful to tell people to install conda, users are supposed to know how to do that. Just give the minimal Python version supported (Python 3.10+ is a safe bet).
To install Python packages give `pip` based command since this is the standard Python package installer.

  * With **cd** command go to directory where you have downloaded /move the python file (use TAB to help fill the mane of directory and **dir** (on windows) / **ls** (linux) to know the list files on the directory)
  * With the peripheral manage **find** the COM of your nRF with the RX code
  * With text file editor **change** the COM  with the write one

>   * **Launch** the script: _python test_radio.py_

Reviewer: If you add `#!/usr/bin/env python` (this is called shebang) and the very beginning of the script and give executable permission to the script, you don't have to use "python" and can just call it directly.

```suggestion
  * **Launch** the script: _test_radio.py_
```

>   * The script will end when you disconnect the nRF with the Rx code and give you the result of the radio communication( CRC, Error, number of packets)

Reviewer: ```suggestion
  * The script ends when you disconnect the nRF with the Rx code and gives you the result of the radio communication( CRC, Error, number of packets)
```
